int32 id
float64 timeout

navig/MsgGlobalPosition begin
navig/MsgGlobalPosition end
float64 thrust